lib/hubspot/meeting.rb

```diff
@@ -0,0 +1,49 @@
+require 'hubspot/utils'
+
+module Hubspot
+  class Meeting
+    #
+    # HubSpot Meeting API
+    #
+    # {https://developers.hubspot.com/docs/api/crm/meetings}
+    #
+    CREATE_MEETING_PATH    = '/crm/v3/objects/meetings'
+    MEETING_PATH           = '/crm/v3/objects/meetings/:meeting_id'
+    ASSOCIATE_MEETING_PATH = '/crm/v3/objects/meetings/:meeting_id/associations/Contact/:contact_id/meeting_event_to_contact'
+
+    class << self
+      def create!(owner_id, meeting_title, meeting_body, start_date_time, end_date_time)
+        body = {
+          properties: {
+            hs_timestamp: DateTime.now.strftime('%Q'),
+            hubspot_owner_id: owner_id,
+            hs_meeting_title: meeting_title,
+            hs_meeting_body: meeting_body,
+            hs_meeting_start_time: start_date_time,
+            hs_meeting_end_time: end_date_time,
+            hs_meeting_outcome: 'SCHEDULED'
+          }
+        }
+        response = Hubspot::Connection.post_json(CREATE_MEETING_PATH, params: {}, body: body )
+        HashWithIndifferentAccess.new(response)
+      end
+
+      def destroy!(meeting_id)
+        Hubspot::Connection.delete_json(MEETING_PATH, {meeting_id: meeting_id})
+        @destroyed = true
+      end
+
+      def destroyed?
+        !!@destroyed
+      end
```

Comment:
heu en fait t'as mélangé un truc de classe et d'instance là
je doute que `Meeting.destroyed?` soit intéressant (en tout cas sans id)
```suggestion
      end
```